sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -22,6 +22,8 @@ import java.net.URI
 import java.sql.Timestamp
 import java.util.{Locale, TimeZone}
 
+import com.google.common.io.Files
```

Comment:
Please fix the import order.
https://github.com/apache/spark/blob/b78130123baba87554503e81b8aee3121666ba91/scalastyle-config.xml#L236-L244

Reply:
thanks